bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/TitleImpl.java

```diff
@@ -125,10 +125,17 @@ public String getType() {
     }
 
     @Override
+    @JsonIgnore
```

Comment:
Don't ignore it, it still needs to be part of the JSON export for compatibility reasons. We will `@JsonIgnore` it in v3.